DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -424,8 +421,7 @@ private function loadOdmDocumentManager(array $documentManager, ContainerBuilder
                 new Reference(sprintf('doctrine_phpcr.%s_session', $documentManager['session'])),
                 new Reference(sprintf('doctrine_phpcr.odm.%s_configuration', $documentManager['name'])),
                 new Reference(sprintf('doctrine_phpcr.%s_session.event_manager', $documentManager['session']))
-            ))
-        ;
+            ));
```

Comment:
imho this was correct. if i chain another call, i don't want to move the `;` around.